app/_kong_plugins/rate-limiting.md

```diff
@@ -11,11 +17,26 @@ faqs:
       Yes, Z is the best, much better than Y.
 
       Oscillation within a space time matrix, with respect to quantum intersection matrices, interwoven on a molecular level with geodesic lattice structures to elicit a persistent linkage between subordinate levels of abstraction.
+related_resources:
+  - text: How to create rate limiting tiers with Rate Limiting Advanced
+    url: https://docs.konghq.com/hub/kong-inc/rate-limiting-advanced/how-to/
+  - text: Rate Limiting Advanced plugin overview
+    url: https://docs.konghq.com/hub/kong-inc/rate-limiting-advanced/
 ---
 
 Rate limit how many HTTP requests can be made in a given period of seconds, minutes, hours, days,
 months, or years. If the underlying service or route has no authentication layer, the Client IP address is
-used. Otherwise, the consumer is used if an authentication plugin has been configured
+used. Otherwise, the consumer is used if an authentication plugin has been configured.
+
+The advanced version of this plugin, [Rate Limiting Advanced](https://docs.konghq.com/hub/kong-inc/rate-limiting-advanced/), provides the ability to apply [mutliple limits in sliding or fixed windows](https://docs.konghq.com/hub/kong-inc/rate-limiting-advanced/#multi-limits-windows).
+
+<!-- example plugin schema renderer below, ideally this will render on the left of the config examples -->
+
+{% schema %}
+type: plugin
+data:
+  name: rate-limiting
+{% endschema %}
```

Comment:
```suggestion
## schema goes here

<!-- example plugin schema renderer below, ideally this will render on the left of the config examples 

{% schema %}
type: plugin
data:
  name: rate-limiting
{% endschema %}
-->
```